warehouse/templates/email/collaborator-added.body.txt

```diff
@@ -0,0 +1,5 @@
+'{{ username }}' has been added as a collaborator with '{{ role }}' role to the PyPI
+project: '{{ project }}' by '{{ submitter }}'.
```

Comment:
I've been taking care to keep these emails at a fixed 80-character limit, which requires a bit of crafting of the template.

For this, we probably want to start out with the same "greeting" every time rather than the username as well.

So maybe something like:

```
A new collaborator has been added to a project you own on PyPI:

  Username: {{ username }}
  Role: {{ role }}
  Collaborator for: {{ project }}
  Added by: {{ submitter }}

If this was a mistake, you can reply to this email directly to communicate with
the PyPI administrators.
```